### PR TITLE
Ensure ClamAV network policy tracks apps when Terraform replaces them

### PR DESCRIPTION
**Pull request by mogul**

### CODE REVIEW

**Review by rahearn — APPROVED** [review]
Approved, though if there is a link to an issue for us to check in the future for when we might be able to (or need to) remove this please add it into this comment before merging.

File: clamav/main.tf
Comment (mogul) [reply clamav/main.tf:45]: Not specifically... They have been migrating resources one at a time when people have the energy.

Although, hmmmm, [there's reference to `rolling` and v3 in this commit](https://github.com/cloudfoundry-community/terraform-provider-cloudfoundry/pull/453)... Maybe they already did it and just didn't update the docs...?

If that's the case, they might have kept the `id_bg` attribute around for backward compatibility and to keep that `venerable` strategy working.

Comment (mogul) [reply clamav/main.tf:45]: [WHY HELLO THERE](https://github.com/Thanhphan1147/terraform-provider-cloudfoundry/blob/93a092cd42258965963de5c1ee4e9aa243b34397/cloudfoundry/managers/v3appdeployers/rolling_strategy.go)

Comment (mogul) [reply clamav/main.tf:45]: https://github.com/cloudfoundry-community/terraform-provider-cloudfoundry/pull/482

**Pull request merged**
